modules/statics/src/coins.ts

```diff
@@ -151,6 +151,7 @@ export const coins = CoinMap.fromCoins([
   ofc('ofcbsv', 'Bitcoin SV', 8, UnderlyingAsset.BSV, CoinKind.CRYPTO),
   ofc('ofceos', 'Eos', 4, UnderlyingAsset.EOS, CoinKind.CRYPTO),
   ofc('ofcetc', 'Ethereum Classic', 18, UnderlyingAsset.ETC, CoinKind.CRYPTO),
+  ofc('ofchbar', 'Mainnet Hedera HBAR', 8, UnderlyingAsset.HBAR, CoinKind.CRYPTO),
```

Comment:
Note: this is "Mainnet Hedera HBAR" instead of just "Hedera HBAR" because that is how the underlying token name is defined on line 124.